agent/php_wrapper.c

```diff
@@ -239,14 +226,6 @@ zval* nr_php_arg_get(ssize_t index, NR_EXECUTE_PROTO TSRMLS_DC) {
       ZVAL_DUP(arg, orig);
     }
   }
```

Comment:
I'm not sure why this block is enclosed in `{`/`}`. These could probably be removed and this logic de-nested.